src/Sylius/Bundle/CoreBundle/Command/InstallCommand.php

```diff
@@ -36,6 +36,10 @@ final class InstallCommand extends AbstractInstallCommand
             'command' => 'setup',
             'message' => 'Shop configuration.',
         ],
+        [
+            'command' => 'jwt_setup',
+            'message' => 'JWT configuration.',
```

Comment:
As we have "**Checking** system requirements", "**Setting** up the database" and "**Installing** assets" I vote to change it to `Configuring JWT token.`... but then we should also change `Shop configuration.` to `Configuring Shop.` 😄 💃  Nevertheless, consistency is the clear to world peace (or at least code peace)